Xext/xres.c

```diff
@@ -870,12 +879,20 @@ ProcXResQueryResourceBytes (ClientPtr client)
     REQUEST(xXResQueryResourceBytesReq);
     REQUEST_AT_LEAST_SIZE(xXResQueryResourceBytesReq);
 
-    ConstructResourceBytesCtx ctx;
-    if (stuff->numSpecs > UINT32_MAX / sizeof(ctx.specs[0]))
-        return BadLength;
```

Comment:
Do we need to keep this check?

Reply:
Shouldn't it be already catched by the REQUEST_FIXED_SIZE() on the top ?

Or do we need to be concerned that this macro could internally cause int overflow ?
It's using uint64_t explicitly, so I believe it should be safe.

Reply:
to force the compiler to multiply on 64bits instead of 32bits ?
ok, added this.

Reply:
Hm, I can’t see the change, did you forget to push it?

Reply:
pushed again.